### PR TITLE
Experimental support for synchronous tasks

### PR DESCRIPTION
**Pull request by vasilev**
It's experimental support for tasks which acted as synchronous.

Usage example:
```
var iron_worker = require('iron_worker');
var worker = new iron_worker.Client();

worker.tasksRun('iron/hello', {}, {}, function (error, result) {
    console.info(result);
})
```
`result` in a callback function will contain a task's stdout.

### CODE REVIEW

**Comment by vasilev** [comment]
@rkononov , @freeman1988 , @alex-litvak Could you please take a look ?

File: src/client.coffee [hunk 1]
```diff
@@ -110,6 +110,14 @@ class Client
         cb(error, body)
     )
 
+  tasksStdout: (taskId, cb) ->
+    @api.tasksStdout(taskId, (error, body) ->
+      if not error?
+        cb(error, body)
+      else
+        cb(error, body)
```
Comment (alex-litvak) [inline src/client.coffee]: Maybe replace the "if - else" block with one line:
cb(error, body)

Comment (vasilev) [reply src/client.coffee]: Just following the style of the surrounding code.

**Comment by vasilev** [comment]
Thank you gentlemen for review !

**Pull request merged**